.github/workflows/CI.yml

```diff
@@ -1,38 +1,28 @@
-# This is a basic workflow to help you get started with Actions
-
-name: CI
-
-# Controls when the action will run. Triggers the workflow on push or pull request
-# events but only for the master branch
-on:
-  push:
-    branches: [ master ]
+name: Test PR and Image
+on: pull_request_target
```

Comment:
should this be 
```
on:
    pull_request_target:
       branches: [ master ]
```

Reply:
sorry I'm looking at https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target
my above suggestion is wrong I think